### ISSUE TITLE
Native ad in androidX squeezed the visit site button 

### ISSUE THREAD

**Issue by androidcodehunter** [googleads/googleads-mobile-android-native-templates#3]
If I convert the xml atributes to androidX, the visit website button get squeezed.

**Comment by sanjaykumarm** [comment]
@androidcodehunter It is may be due to long text, can you please paste before and after screenshots here ?

**Comment by androidcodehunter** [comment]
I cannot paste it now because my admob account has been deactivated but I
say this problem.

**Comment by sanjaykumarm** [comment]
@androidcodehunter Any specific reason for deactivated your account ? Which policy violation ? It may help other developers as well if you could paste email here or write full details so that other developers who are implementing native ad will take care of it.

**Comment by androidcodehunter** [comment]
I don't know specific reason cause Google don't want to disclose the reason
specifically. But Google said that because of invalid activity my account
disabled.

**Comment by sanjaykumarm** [comment]
@androidcodehunter Oh I am really sad that your account disabled but it is one of the biggest problem of Google, they are not giving warning or some notice within which we can fix the problem.

**Comment by androidcodehunter** [comment]
Yes, without any warning they disabled my account. I was so cautious but no
avail. Even I don't know the clear reason. I am using Facebook Audience
Network(FAN) now.

Thanks,
Sharifur

**Comment by hemantv** [comment]
![Screenshot_20191013-203525](https://user-images.githubusercontent.com/31650155/66718011-a9556c00-edfc-11e9-9cda-46804c5fba75.png)

Having the same issue. Any solution to it?

**Comment by androidcodehunter** [comment]
I don't have any solution for this specific problem.

**Comment by AmJay** [comment]
So its been 6 months now, and this problem still remains.

**Comment by sanjaykumarm** [comment]
@AmJay Please send me your email here, I will help you.

**Comment by the-solitary-observer** [comment]
> ![Screenshot_20191013-203525](https://user-images.githubusercontent.com/31650155/66718011-a9556c00-edfc-11e9-9cda-46804c5fba75.png)
> 
> Having the same issue. Any solution to it?

I am having the exact same issue.

**Comment by davidecutrupi** [comment]
**Add these lines in your button tag:**
android:padding="0dp"
android:insetTop="0dp"
android:insetBottom="0dp"

**(in my case to change the background of the button I used backgroundTint instead of background)**

**Comment by wangjiejacques** [comment]
I just replace the button with TextView.
